_sass/components/_navigation.scss

```diff
@@ -0,0 +1,61 @@
+.c-navigation {
+  position: absolute;
+  width: 100%;
+
+  .c-navigation__logo {
+    width: 4rem;
+    @media (min-width:$bp-sm){
+    width:6rem;
+  }
+  }
+
+.c-navigation__container-mobile {
+  width: 80%;
+  margin: 3rem auto 0;
+
+.c-navigation__mobile{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  @media (min-width:68.375rem){
```

Comment:
Could this breakpoint, 68.375rem, be a sass variable as well? Such as `$nav-breakpoint` or something like that?